prebid-server/pbs-modules/rules-engine.md

```diff
@@ -0,0 +1,805 @@
+
+### Module Account-Level Configuration
+
+These are the full set of attributes that can be present in the account-level configuration:
+
+{: .table .table-bordered .table-striped }
+| Property | Scope | Type | Description | Example |
+|----------|-------|------|-------------|---------|
+| enabled | optional | boolean | Whether the module as a whole is enabled. Allows for easy temporary deactivation. Defaults to true. | true |
+| generateRulesFromBidderConfig | optional | boolean | (PBS-Go only) If true, the module automatically generates rules based on 'geoscope' section of the bidder YAML files to include or exclude that bidder from certain countries. Defaults to true. | false |
+| ruleSets[] | required | array of objects | One or more independent sets of rules. | |
+| ruleSets[]<wbr>.enabled | optional | boolean | Indicates whether this ruleset is active. Defaults to true. Allows easy toggling for troubleshooting. | true |
+| ruleSets[]<wbr>.timestamp | optional | string | String indicates when this ruleset was last updated. Useful in optimization and troubleshooting. Format is ISO 8601. | "20250101T00:00:00" |
+| ruleSets[]<wbr>.stage | required | string | Which module stage this object applies to. Initially, the only value allowed is "processed auction" | "processed auction" |
+| rulesets[]<wbr>.name | optional | string | Just for human readability. | "remove-bidder-by-datacenter" |
+| ruleSets[]<wbr>.modelGroups[] | required | array of objects | This is where the rule details are stored. It's an array in case there's a desire to A/B test different rule configs. Model groups work here like they do for [floors](/dev-docs/modules/floors.html#schema-2). | |
+| ruleSets[]<wbr>.modelGroups[]<wbr>.weight | optional | integer | The relative weight of this specific modelGroup entry. Only one array entry within this modelGroup will be chosen. Default weight is 1, and they must range from 1 to 100. | 10 |
+| ruleSets[]<wbr>.modelGroups[]<wbr>.version | optional | string | In case the machine learning system would like to track the specifics of an experiment. | "123" |
+| ruleSets[]<wbr>.modelGroups[]<wbr>.default[] | optional | array of objects | The default results object to be used if anything goes wrong while traversing the rule tree or if there aren't any rules defined. | |
+| ruleSets[]<wbr>.modelGroups[]<wbr>.default[]<wbr>.function | optional | string | Which results function to call. | "logAtag" |
+| ruleSets[]<wbr>.modelGroups[]<wbr>.default[]<wbr>.args | optional | object or array of objects | Which arguments to pass to the function. | |
+| ruleSets[]<wbr>.modelGroups[]<wbr>.schema[] | optional | array of objects | Array of functions used to define the tree. | |
+| ruleSets[]<wbr>.modelGroups[]<wbr>.schema[]<wbr>.function | optional | string | Which schema function to call. | "deviceCountry" |
+| ruleSets[]<wbr>.modelGroups[]<wbr>.schema[]<wbr>.args | optional | array | Arguments to pass to the schema function. | [["USA","CAN"]] |
```

Comment:
`args` is always an object

The example here should read: `{"countries": ["USA", "CAN"]}`